docs/quickstart/create-a-local-test-network.md

```diff
@@ -17,24 +17,26 @@ The Avalanche Network Runner repository is hosted at [https://github.com/ava-lab
 
 That repository's README details the tool.
 
-Clone the repository with:
+To download a binary for the latest release, run:
 
-```bash
-git clone https://github.com/ava-labs/avalanche-network-runner.git
+```shell
+curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-network-runner/main/scripts/install.sh | sh -s
 ```
 
-There are also binary releases ready to use at [releases](https://github.com/ava-labs/avalanche-network-runner/releases). You can download and install it on to your computer.
+The script installs the binary inside the `~/bin` directory.
```

Comment:
I assume `~/bin` gets created if it doesn't exist, so a small note about that doesn't hurt?